### ISSUE TITLE
Error compiling html.js when using wrapRootElement in gatsby-browser.js and gatsby-ssr.js with select firebase imports

### ISSUE THREAD

**Issue by BenAOlson** [alexluong/gatsby-packages#19]
As mentioned in [this comment on another issue](https://github.com/alexluong/gatsby-packages/issues/17#issuecomment-594214515), there is an error compiling html.js after running either `gatsby develop` or `gatsby build` when `wrapRootElement` is used in conjunction with select firebase package imports in `gatsby-browser.js` and `gatsby-ssr.js`. Specifically, at least `messaging`, `analytics`, and `performance` all cause this error when imported, whereas `firestore` and `functions` (for example) do not cause the error.

[Github repo demonstrating the problem](https://github.com/BenAOlson/gatsby-firebase-bug-demo)

**Comment by BenAOlson** [comment]
It occurs to me that it's not obvious why those firebase packages would need to be imported to `gatsby-ssr.js` anyway. I'm fairly new to Gatsby, but it seems like it would be fine to just import them in `gatsby-browser.js` since firebase only needs to be used during runtime, right?

I've tried building my project without any firebase imports in `gatsby-ssr.js` and everything seems to work correctly. Maybe the docs just need to be updated?

**Comment by alexluong** [comment]
@BenAOlson That's a valid point. If you don't include the Firebase imports in `gatsby-ssr.js`, everything works as expected?

Would you be interested in making a PR in the docs?

**Comment by BenAOlson** [comment]
Yes, as far as I can tell everything is working correctly of the things I have used so far in both development and from a full build.
